Gemfile

```diff
@@ -121,34 +110,48 @@ unless ENV['APPLIANCE']
   end
 end
 
-# Assets from rails-assets.org
-source "https://rails-assets.org" do
-  gem "rails-assets-angular",                         "~>1.4.3"
-  gem "rails-assets-angular-animate",                 "~>1.4.3"
-  gem "rails-assets-angular-mocks",                   "~>1.4.3"
-  gem "rails-assets-angular-patternfly",              "~>2.8.0"
-  gem "rails-assets-angular-sanitize",                "~>1.4.3"
-  gem "rails-assets-bootstrap-datepicker",            "~>1.4.0"
-  gem "rails-assets-bootstrap-filestyle",             "~>1.2.1"
-  gem "rails-assets-bootstrap-hover-dropdown",        "~>2.0.11"
-  gem "rails-assets-bootstrap-select",                "~>1.7.3"
-  gem "rails-assets-c3",                              "~>0.4.10"
-  gem "rails-assets-himdel--jquery.observe_field",    "~>0.1.0"
-  gem "rails-assets-jasmine-jquery",                  "~>2.1.1"
-  gem "rails-assets-jquery-1.8",                      "~>1.8.3"
-  gem "rails-assets-jquery-ujs",                      "~>1.1.0"
-  gem "rails-assets-jqueryui",                        "~>1.9.2"
-  gem "rails-assets-bootstrap-switch",                "~>3.3.2"
-  gem "rails-assets-angular-bootstrap-switch",        "~>0.4.1"
-  gem "rails-assets-kubernetes-topology-graph",       "= 0.0.22"
-  gem "rails-assets-moment",                          "~>2.10.3"
-  gem "rails-assets-moment-strftime",                 "~>0.1.5"
-  gem "rails-assets-moment-timezone",                 "~>0.4.0"
-  gem "rails-assets-numeral",                         "~>1.5.3"
-  gem "rails-assets-slickgrid",                       "~>2.1.0"
-  gem "rails-assets-spin.js",                         "~>2.3.2"
-  gem "rails-assets-sprintf",                         "~>1.0.3"
-  gem "rails-assets-xml_display",                     "~>0.1.1"
+# Frontend dependencies does not required in production
```

Comment:
typo:  "does not required"

Are these required on appliances?  If not, we can put them in a unless `ENV['APPLIANCE']`.